fabric-enchantment-api-v1/src/main/java/net/fabricmc/fabric/api/enchantment/v1/EnchantmentEvents.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.enchantment.v1;
+
+import net.minecraft.enchantment.Enchantment;
+import net.minecraft.item.ItemStack;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+import net.fabricmc.fabric.api.util.TriState;
+
+/**
+ * Events to intercept the vanilla enchantment logic.
+ *
+ * @author Vaerian (vaeriann@gmail.com or @Vaerian on GitHub).
```

Comment:
Author tag isn't required, we can just `git blame` in the worst case

Reply:
I added it to all of the files just because I know there's a history of modules getting merged in and then the burden of updating them falls on someone who has no idea how it works. The idea was to put my contact information in there so that if someone needs information about how the module works, or needs it updated I can take a look at it so that the burden doesn't fall solely on someone else. It can be removed however.

Reply:
I dont see a big issue with it being there